FEEDBACK.md

```diff
@@ -0,0 +1,334 @@
+
+Using a dialog is a soft push for feedback. It lets the user continue with their work with a minimal amount of intrusion. The user can opt in to responding, but is not required to. A good use case for this behavior is when a user used a feature where the expected outcome is predictable, possibly because they have used it multiple times before. For example: Uploading a file, switching to a different view of a visualization, visiting a specific page or manipulating some data.
+
+The default feedback UI behavior is a dialog placed at the bottom right of the users viewport.
+
+```javascript
+position: {
+  type: "DIALOG",
+  placement: "top-left" | "top-right" | "bottom-left" | "bottom-right"
+}
+```
+
+#### Popover
+
+A popover that is anchored relative to a DOM-element (typically a button). Can be dismissed with an outside click or pressing a close button.
+
+You can use the popover mode to implement your own button to collect feedback manually.
+
+```javascript
+position: {
+  type: "POPOVER",
+  anchor: DOMElement
+}
+```
+
+Popover feedback button example:
+
+```html
+<button id="feedbackButton">Tell us what you think</button>
+<script>
+  const button = document.getElementById("feedbackButton");
+  button.addEventListener("click", (e) => {
+    bucket.requestFeedback({
+      featureId: "bucket-feature-id",
+      userId: "your-user-id",
+      title: "How do you like the popover?",
+      position: {
+        type: "POPOVER",
+        anchor: e.currentTarget,
+      },
+    });
+  });
+</script>
+```
+
+## Internationalization (i18n)
+
+The Feedback UI currently only comes in English. You can supply your own translation keys that will be used insude the UI by passing an object with your own translation keys.
+
+See [default english localization keys](./src/feedback/config/defaultTranslations.tsx) for a reference of what translation keys can be supplied.
+
+### Static language configuration
+
+If you know the language at page load, you can configure your translation keys while initializing the Bucket SDK:
+
+```javascript
+bucket.init("my-tracking-key", {
+  feedback: {
+    ui: {
+      translations: {
+        // Your translation keys
+      },
+    },
+  },
+});
+```
+
+### Runtime language configuration
+
+If you only know the users language after the page has loaded, you can provide translation keys in the two cases where you open the feedback UI:
+
+### Manual feedback collection
+
+```javascript
+bucket.requestFeedback({
+  ... // Other options
+  translations: {
+    // your translation keys
+  }
+})
+```
+
+### Automated feedback collection
+
+When you are collecting feedback through the Bucket automation, you can intercept the default prompt handling and override the defaults.
+
+If you set the prompt question in the Bucket app to be one of your own translation keys, you can even get a translated version of the question you want to ask your customer in the feedback UI.
```

Comment:
I suggest:

If you set the prompt question in the Bucket app to be one of your own translations, you even get a translated version of the question you want to ask your customer in the feedback interface.

Reply:
```suggestion
If you set the prompt question in the Bucket app to be one of your own translations, you even get a translated version of the question you want to ask your customer in the feedback interface.
```

Reply:
Discarding suggestion. In this case `translation key` is actually what they want to be setting